README.md

```diff
@@ -27,14 +27,24 @@ learning frameworks.
 ## Key Features
 
 - Support of various compression algorithms, applied during a model fine-tuning process to achieve a better performance-accuracy trade-off:
-  
+
+### Compression-Aware Training Algorithms  
+
 |Compression algorithm|PyTorch|TensorFlow|
 | :--- | :---: | :---: |
 |[Quantization](./docs/compression_algorithms/Quantization.md) | Supported | Supported |
-|[Mixed-Precision Quantization](./docs/compression_algorithms/Quantization.md#mixed_precision_quantization) | Supported | Not supported |
-|[Binarization](./docs/compression_algorithms/Binarization.md) | Supported | Not supported |
-|[Sparsity](./docs/compression_algorithms/Sparsity.md) | Supported | Supported |
-|[Filter pruning](./docs/compression_algorithms/Pruning.md) | Supported | Supported |
+|[Mixed-Precision Quantization](./docs/compression_algorithms/Quantization.md#mixed_precision_quantization) | Supported | Not supported | Not supported |
+|[Binarization](./docs/compression_algorithms/Binarization.md) | Supported | Not supported | Not supported |
+|[Sparsity](./docs/compression_algorithms/Sparsity.md) | Supported | Supported | Not supported |
+|[Filter pruning](./docs/compression_algorithms/Pruning.md) | Supported | Supported | Not supported |
+
+### Post-Training Compression Algorithms
```

Comment:
It looks good to me. Please place it before and rename "Compression-Aware Training Algorithms " to "Training-time Compression Algorithms" so that it makes aligned with "Post-Training Compression Algorithms".